ballot/models.py

```diff
@@ -966,7 +991,10 @@ def delete_all_ballot_items_for_voter(self, voter_id, google_civic_election_id):
         }
         return results
 
-    def retrieve_ballot_items_for_election(self, google_civic_election_id, state_code=''):
+    @staticmethod
+    def retrieve_ballot_items_for_election(
```

Comment:
We only need to break variables onto a second line when there are enough incoming variables to cause a line wrap.